stock_measuring_device/wizard/measuring_wizard_line.py

```diff
@@ -14,9 +14,9 @@ class MeasuringWizardLine(models.TransientModel):
     name = fields.Char("Packaging", readonly=True)
     qty = fields.Float("Quantity")
     max_weight = fields.Float("Weight (kg)", readonly=True)
-    # this is not a typo:
-    # https://github.com/odoo/odoo/issues/41353#issuecomment-568037415
-    lngth = fields.Integer("Length (mm)", readonly=True)
+    # this is renamed to match with a change in product_packaging_dimension
+    # https://github.com/OCA/product-attribute/commit/24c773cb7c0099bf2f6ac11509e1aaa6781cd03b
```

Comment:
You can drop this comment IMO, it's not a valuable information for upcoming migrations (but it's a great info for reviewers if you do it as a comment when you open the PR).